Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -177,6 +177,22 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         return;
     }
 
+    if (createOptions.signal) {
+        createOptions.signal->addAlgorithm([weakThis = WeakPtr { *this }](JSC::JSValue) mutable {
+            if (!weakThis)
+                return;
+            weakThis->m_client->cancel([weakThis = WTFMove(weakThis)] () mutable {
+                if (!weakThis)
+                    return;
+                weakThis->m_isCancelling = true;
```

Comment:
This only gets called after the cancel is complete, this needs to be inside the signal block (happens whenever the rp calls the abortSignal) but outside the cancel completion handler (happens when the cancel is complete.)